src/main/java/edu/harvard/iq/dataverse/api/MakeDataCountApi.java

```diff
@@ -110,7 +116,7 @@ public Response addUsageMetricsFromSushiReport(@PathParam("id") String id, @Quer
 
     @POST
     @Path("/addUsageMetricsFromSushiReport")
-    public Response addUsageMetricsFromSushiReportAll(@PathParam("id") String id, @QueryParam("reportOnDisk") String reportOnDisk) {
+    public Response addUsageMetricsFromSushiReportAll(@QueryParam("reportOnDisk") String reportOnDisk) {
```

Comment:
I'm confused. Is this a backward-incompatible change? If so, is it necessary? If so, can you document it in the API changelog? The one from 6.1: https://guides.dataverse.org/en/6.1/api/changelog.html

Reply:
Ah, I talked to @stevenwinship and he pointed out it was unused. Resolving!